shell/common/gin_converters/net_converter.h

```diff
@@ -112,6 +114,35 @@ struct Converter<net::RedirectInfo> {
                                    const net::RedirectInfo& val);
 };
 
+template <typename K, typename V>
+struct Converter<std::vector<std::pair<K, V>>> {
```

Comment:
There is already a converter for `std::vector` so you only need to write a converter for `std::pair`.

Reply:
Could you please point a direction for me. Should I change the headers to an array if I write a converter for std::pair instead? Currently the headers is an Object, I'm not sure It'll work or not. Thank you.

Reply:
Sorry I did not notice the headers is Object, in this case this converter is fine.